src/riak_kv_get_core.erl

```diff
@@ -284,6 +288,36 @@ num_pr(GetCore = #getcore{num_pok=NumPOK, idx_type=IdxType}, Idx) ->
             GetCore
     end.
 
+%% @private Print a warning if objects are not equal. Only called on case of no read-repair
+%% This situation could happen with pre 2.1 vclocks in very rare cases. Fixing the object
+%% requires the user to rewrite the object in 2.1+ of Riak. Logic is disabled when using
+%% a defined version of aae hashing as it's no longer needed.
+maybe_log_old_vclock(Results) ->
+    case riak_core_capability:get({riak_kv, object_hash_version}, undefined) of
```

Comment:
Oh wait, so actually I misread the code there. The comment up above this function says "logic is disabled when using a defined version of aae hashing as it's no longer needed," but it looks like we actually disable it when we're *not* using a defined version of aae hashing, right? Is this actually what we want to do though? Feels like something is backwards here but maybe it's just my brain hitting late Friday afternoon and starting to stall out on me ;)